src/twisted/newsfragments/12130.removal

```diff
@@ -0,0 +1 @@
+async keyword argument was deprecated in favor of isAsync in twisted.conch.manhole (ManholeInterpreter.write and Manhole.add) and in twisted.main.imap4.IMAP4Server.sendUntaggedResponse in Twisted 18.9.0 and has been removed
```

Comment:
The release notes are good.

Maybe is best to also add explicitit names for the manhole API, in the case someone is searching for `addOutput` method.

The text is formated as documented at https://docs.twisted.org/en/stable/development/dev-process.html#release-notes-management

starting with the affected API.

```suggestion
twisted.conch.manhole.ManholeInterpreter.write, twisted.conch.manhole.ManholeInterpreter.addOutput, twisted.mail.imap4.IMAP4Server.sendUntaggedResponse `async` argument, deprecated since 18.9.0, has been removed.
```

Reply:
@adiroiban Thank you for the review, I've added your suggested changes and rebased my branch so it is still only 1 commit. 😄